### PR TITLE
Display html file after successfull discord link

### PR DESCRIPTION
**Pull request by TheodosiouTh**
Solves #24

### CODE REVIEW

File: src/pages/account-linked.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Successful link</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@1,700&display=swap');
```
Comment (zkldi) [inline src/pages/account-linked.html:9]: I think this should be changed to `'https://fonts.googleapis.com/css2?family=Poppins&display=swap'`

That's the font we use on the rest of the site, and the bold/italics are bit much in my opinion.

File: src/pages/account-linked.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Successful link</title>
```
Comment (zkldi) [inline src/pages/account-linked.html:7]: Could maybe change this to something like `Discord Authentication Successful`. Sounds cooler.

**Review by zkldi — CHANGES_REQUESTED** [review]
Added a couple nitpicks, but other than that this looks really cool! Thank you!

File: src/pages/account-linked.html (hunk 2 above)
Comment (Puffycheeses) [reply src/pages/account-linked.html:7]: `Discord Authentication Successful` would work better in this context

**Review by Puffycheeses — APPROVED** [review]

File: src/pages/account-linked.html (hunk 1 above)
Comment (TheodosiouTh) [reply src/pages/account-linked.html]: I updated the import according to your suggestion.

File: src/pages/account-linked.html (hunk 2 above)
Comment (TheodosiouTh) [reply src/pages/account-linked.html]: Yeah `Discord Authentication Successful` sounds much better.

**Comment by TheodosiouTh** [comment]
@zkldi @Puffycheeses  I updated the code with the requested changes.

**Comment by zkldi** [comment]
Looking good! If you don't mind me asking by the way - how did you come across this gh issue?

**Comment by TheodosiouTh** [comment]
@zkldi I found it through the  `help wanted  [search](https://github.com/search?q=label%3A%22help+wanted%22). I search around this page as I want to contribute more to Open source.

**Comment by zkldi** [comment]
> @zkldi I found it through the `help wanted` [search](https://github.com/search?q=label%3A%22help+wanted%22). I search around this page as I want to contribute more to Open source.

Very cool stuff! Thank you for your contributions. I'll merge into main now!

**Pull request merged**